packages/dependency-extraction-webpack-plugin/test/fixtures/polyfill-magic-comment/webpack.config.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Internal dependencies
+ */
+const DependencyExtractionWebpackPlugin = require( '../../..' );
+
+module.exports = {
+	plugins: [
+		new DependencyExtractionWebpackPlugin( {
+			requestToExternalModule( request ) {
+				return request.startsWith( '@wordpress/' );
+			},
+		} ),
```

Comment:
I think the plugin with defaults should work fine:

```suggestion
new DependencyExtractionWebpackPlugin(),
```